detox/src/artifacts/instruments/ios/SimulatorInstrumentsPlugin.js

```diff
@@ -14,6 +14,10 @@ class SimulatorInstrumentsPlugin extends InstrumentsArtifactPlugin {
 
     if (this.testRecording) {
       event.launchArgs['recordingPath'] = this.testRecording.temporaryRecordingPath;
+      event.launchArgs['samplingInterval'] =
+        this.api.userConfig.samplingInterval
+          ? Number(this.api.userConfig.samplingInterval) / 1000.0
+          : 0.25;
```

Comment:
Looks a bit disturbing, in a sense that you put an inline logic with divisions and default values here, instead of using `prepareSamplingInterval`, :confused: ? Should not the same logic be applied if you send JSON over Websocket in SimulatorInstrumentsPlugin?